Build document failed in a new workspace

I see this error when I try to build document from a new workspace.

$ cd XRT/src/runtime_src/doc/
$ make
/lib/modules/4.4.0-128-generic/build/scripts/kernel-doc -rst ../driver/xclng/include/mgmt-ioctl.h > core/mgmt-ioctl.rst
/bin/sh: core/mgmt-ioctl.rst: No such file or directory
Makefile:22: recipe for target 'core/mgmt-ioctl.rst' failed
make: *** [core/mgmt-ioctl.rst] Error 1

--- a/src/runtime_src/doc/Makefile
+++ b/src/runtime_src/doc/Makefile
@@ -30,7 +30,10 @@ core/xclhal2.rst : $(XRTSRCROOT)/driver/include/xclhal2.h
 core/ert.rst : $(XRTSRCROOT)/driver/include/ert.h
 	$(KERNELDOC) -rst $< > $@
 
-$(OUTDIR)/index.html: $(RST)
+core : 
+	mkdir -p core/
+
+$(OUTDIR)/index.html: core $(RST)
 	mkdir -p $(OUTDIR)
 	$(SPHINX) $(SRCDIR) $(OUTDIR)
 

